comments/request.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+import json
+from models import Comment
+
+def create_comment(new_comment):
+    '''Reader/Author can add a comment to an author's post'''
+    with sqlite3.connect("./rare.db") as conn:
+        db_cursor = conn.cursor()
+
+        db_cursor.execute("""
+        INSERT INTO Comments
+            ( post_id, author_id, content )
```

Comment:
This does not meet the feature requirements specified in the issue ticket! Please review